website/blog/2023-12-06-release-1.6.md

```diff
@@ -0,0 +1,250 @@
+
+
+### Command-Line Tools Configuration: Compose and Kubectl
+Configuring and managing your setup is getting easier with the addition of a new section in the Settings to manage Command-line tools. In Podman Desktop, extensions can bring dependencies and tools that are going to be necessary for working properly. 
+
+Two Command-Line tools are getting registered at the moment Compose and kubectl:
+- Compose-cli is registered by the Compose extension. 
+- kubectl is a dependency needed with Minikube extension.
+
+![cli-tools](img/podman-desktop-release-1.6/cli-tools.png)
+
+From the settings you can see the command-line tools that are installed, you can see the version - and when a new version is available, you'll get a small notification to allow you easily updating that dependency. 
+
+![Compose-Update](img/podman-desktop-release-1.6/compose-update.png)
+
+
+### Kubernetes Contexts Manager
+We are introducing a new screen available from the settings allowing to easily manage your Kubernetes contexts. Podman Desktop was already providing the handy context switcher available from the status bar, but when you get to work with multiple Kubernetes environments, it's not uncommon to ended with a big and long list of Kubernetes contexts.
+
+The new Kubernetes Contexts screen allows you to easily see all your registered Kubernetes contexts. You can use that screen to cleanup your registered contexts, and also to define the context you want to be using by default.
+
+![Kubernetes Contexts List](img/podman-desktop-release-1.6/kubernetes-contexts.png)
+
+
+### Editable Podman Machine 
+A Podman machine is a virtual environment specifically designed to run Podman containers on Mac and Windows. It allows users to manage and operate containerized applications in an isolated and controlled setting. When creating a Podman machine, you configure its settings: memory, CPU(s) and disk Size. 
+
+We've got feedback from some of you, interested by the ability to reconfigured your Podman machine on the go. This is now possible and particularly useful when you started with an environment and need to scale it up base on new needs and containers you would like to run in your Podman environment. 
+
+<ReactPlayer playing controls url="https://github.com/containers/podman-desktop/assets/1636769/91150767-58a9-47b5-abbc-58d2d50f4fca.gif" width='100%' height='100%' />
+
+You'll also notice, we improved the sliders to configure the Podman machine's options - and we also introduced a way to enter numeric values directly. 
+
+
+### Tabs/Filters for Containers and Pods
+Being able to quickly identify the containers and the pods you are working with is critical when you iterating on the development of your application. For this reason, we added filters at the top of the lists of Containers and Pods that allow you to easily see either see all the containers/pods, only those who are running or only those who are stopped.
```

Comment:
'you iterating' -> 'you are iterating'

'running or' -> 'running, or ' - I like Oxford commas, you don't have to agree. :)